gitpod-io/gitpod
# Start dockerd without experimental flag

## Description
Start dockerd in the workspace without --experimental flag. We originally needed this to support rootless, but it graduated in v20.10

## Related Issue(s)
n.a.

## How to test
- [Browse to preview environment](https://fo-dockerd-exp-flag.staging.gitpod-dev.com/workspaces)
- Open workspace
- Run `docker run busybox sh`
-> this should just work

## Release Notes
```release-note
None
```
